frontend/src/app/shared/components/storages/storage/storage.component.html

```diff
@@ -1,6 +1,18 @@
 <ng-container
   *ngIf="resource"
 >
+  <div class=op-tab-content--header>
+    <span class="op-tab-content--header-icon spot-icon spot-icon_nextcloud-circle op-files-tab--icon op-files-tab--icon_nextcloud"></span>
+    <span class="op-tab-content--header-text" [textContent]="storage.name"></span>
+    <button
+      class="op-tab-content--header-action spot-link"
+      [title]="text.openStorage()"
```

Comment:
this should also be available as an aria attribute, titles are not read by screenreaders afaik

Reply:
hmm, I thought, on most interaction elements title is interpreted. Checking accessibility tab in firefox gives this:

![image](https://user-images.githubusercontent.com/38206611/200832396-26f0d00b-0cf2-404e-970e-942c7606a584.png)

the title appears within the description. Is there an aria attribute you would add additionally?

Reply:
There's a ton of conflicting information surrounding this, mostly because what exactly gets read out loud depends on the screenreader software, OS, and browser. Safest bet is to have both.

Reply:
https://dev.opera.com/articles/ux-accessibility-aria-label/#accessible-name-calculation
https://accessible360.com/accessible360-blog/use-aria-label-screen-reader-text/
https://pauljadam.com/demos/title-aria-label.html

Reply:
so, I could add `aria-label` in addition. From what I was reading now, seems to be the best bet.